models.py

```diff
@@ -152,6 +152,13 @@ def get_full_recipe_from_id(cls, id):
         recipe['ingredients'] = ingredients
         return recipe
 
+    @classmethod
+    def return_recipe_lengths(cls, ids):
```

Comment:
What's `cls`? Consider renaming.

Reply:
cls is Pythons standard usage (it might even be required) for the first parameter passed to any classmethod. It's short for class, (which is a reserved Keyword) and represents the entire class.

I'm not sure if it's required, but if it's not it's standard practice.